shim.c

```diff
@@ -1226,7 +1226,7 @@ static EFI_STATUS handle_image (void *data, unsigned int datasize,
 		return efi_status;
 
 	/* Measure the binary into the TPM */
-	tpm_log_pe((EFI_PHYSICAL_ADDRESS)(UINTN)data, datasize, sha1hash, 9);
+	tpm_log_pe((EFI_PHYSICAL_ADDRESS)(UINTN)data, datasize, sha1hash, 4);
 
```

Comment:
I'd like clarification from https://github.com/mjg59 that this is what he actually intended, since "closer" seems ambiguous, but I think probably yes.